jwcrypto/jwk.py

```diff
@@ -618,8 +625,8 @@ def get_curve(self, arg):
         :raises InvalidJWKValue: if the curve names is invalid.
         """
         k = self._key
-        if self._params['kty'] != 'EC':
-            raise InvalidJWKType('Not an EC key')
+        if self._params['kty'] not in ['EC', 'OKP']:
+            raise InvalidJWKType('Not an EC or OKP key')
```

Comment:
fit the comment above too please.

Reply:
It now reads:
```
:raises InvalidJWKType: the key is not an EC or OKP key.
```
Is that what you wanted?